src/scss/utilities/_spacing-helpers.scss

```diff
@@ -19,7 +19,7 @@ $space-prefixes: (
   @each $attr-short, $attr-long in $space-prefixes {
     @each $value in $space-values {
       .#{$attr-short}-#{$value} {
-        #{$attr-long}: #{$value}#{"px"};
+        #{$attr-long}: #{$value}#{"px"} !important;
```

Comment:
important is not usually recommended maybe we can increase css priority if something needs overriding?

Reply:
yea good point. the !important is very ugly but had to override some out of the box angular material css. will investigate a heavier css selector perhaps as that would be cleaner

Reply:
let me know if you want to override in component itself or revise this later as in the grand scheme of things is not a show stopper.

Reply:
!important removed and pr updated